net.xy.ykhwong.dosbox-daum.yml

```diff
@@ -0,0 +1,92 @@
+app-id: net.xy.ykhwong.dosbox-daum
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: dosbox
+build-options:
+  strip: true
+
+finish-args:         # flatpak permissions
+  - --device=all     # needed for OpenGL, gamepads and serial/parallel
+  - --share=ipc      # needed for X11
+  - --share=network  # Needed for IPX and serial over TCP/IP
+  - --socket=x11     # default to X11, as Gnome does not provide SSD
+  - --socket=pulseaudio
+  - --filesystem=home
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/share/aclocal'
+  - '/share/man'
+  - '*.la'
+  - '*.a'
+  - '/share/doc'
+
+modules:
+  - "shared-modules/glu/glu-9.json"
+  - "shared-modules/glew/glew.json"  
+  - name: SDL-1.2.15-hq
+    buildsystem: simple
+    build-commands:
+      - sh autogen.sh
+      - ./configure --prefix=/app/
+      - make #-e CXXFLAGS="-std=c++0x -stdlib=libstdc++"
+      - make install
+    sources:
+      - type: git
+        url: https://github.com/ebenbruyns/sdl1.2.15-hq.git
+        branch: main #  tag: release-1.2.15
+  - "shared-modules/SDL/SDL_net-1.2.8.json"
+  - name: libtbb
+    buildsystem: simple
+    build-commands:
+      - make -e CXXFLAGS="-std=c++11" # -stdlib=libstdc++" 
+      - install -m755 -t /app/lib/ build/linux_intel64_gcc_*_release/*
+      - mkdir /app/include/tbb
+      - mkdir /app/include/tbb/machine /app/include/tbb/compat /app/include/tbb/internal  
+      - mkdir -p /app/include/serial/tbb  
+      - install -t /app/include/tbb/ include/tbb/*.h
+      - install -t /app/include/serial/tbb include/serial/tbb/*
+      - install -t /app/include/tbb/compat include/tbb//compat/*
+      - install -t /app/include/tbb/machine include/tbb/machine/*
+      - install -t /app/include/tbb/internal include/tbb/internal/*
```

Comment:
```suggestion
      - install -D -t /app/include/tbb/ include/tbb/*.h
      - install -D -t /app/include/serial/tbb include/serial/tbb/*
      - install -D -t /app/include/tbb/compat include/tbb//compat/*
      - install -D -t /app/include/tbb/machine include/tbb/machine/*
      - install -D -t /app/include/tbb/internal include/tbb/internal/*
```

Reply:
Does that actually work? I thought I tried the -D switch